### PR TITLE
Allow angled headers without images, and clean up about page

### PR DESCRIPTION
**Pull request by mirisuzanne**
strange bird

- [Trello card](https://trello.com/c/P8xVB0wo/52-i-want-a-birds-list-page)
- [Preview](https://deploy-preview-32--oddleventy.netlify.com/)

## Description
- [X] Description is in Trello card
- There are a few other bits of related cleanup that happened…

## Impacted areas of website
- About page
- Any page without an image now uses the angled layout by default

### CODE REVIEW

**Comment by wlonk** [comment]
(Just gotta comment on the INTENSE animal pic!)

File: src/scss/layout/_header.scss [hunk 1]
```diff
@@ -38,3 +38,14 @@
     --edge: 25%;
   }
 }
+
+[data-header='angle'] {
+  display: grid;
+  grid-template-columns: var(--typeset);
```
[the remark this answers is not in the document]
Comment (stacyk) [reply src/scss/layout/_header.scss]: narrow-2

**Comment by mirisuzanne** [comment]
@stacyk @SondraE I think I addressed your reviews that I could here.

- the page grid template (formerly `--typeset`, now `--page-columns`) is smarter about when/how the `wide` column is added
- the fit-grid template (used for birds) is now smarter about not overflowing the page

A few things could happen in new stories/branches:
- We can remove whitespace from svg faces when we fix their eyes
- We can decide if bird detail pages use a photo or the face. Right now we have images for some people, but not everyone.

**Comment by stacyk** [comment]
Looks awesome. Not important this moment, but between 608 and 639ish px the intro on the homepage gets cut off. Wasn't sure if the rotating could wait until that 640ish mark perhaps?
Screen Shot 2020-02-27 at 1 24 01 PM

**Comment by mirisuzanne** [comment]
@SondraE 
>     * The emeritus date ranges look like they are using an em dash; should that be an en dash?

fixed!

>     * I expect the top nav to stay active when I visit a detail page (e.g. I thought "About" would stay active when I viewed Stacy's author detail page.) Is there a reason not to do that?

We aren't doing this sort of "nested" navigation anywhere right now – the nav is only active when you are on the page it links to. If we need activation based on subnav, that should be a new story with clear rules about what is "under" what.

@stacyk I think the media-queries all line up now.

**Pull request merged**